doc/set_version.sh

```diff
@@ -0,0 +1,3 @@
+export DOC_VERSION=2024.3
```

Comment:
Can we get this info from library itself?  @ethanglaser  - any thoughts here?

Reply:
In most cases were are calling this function to get onedal version - https://github.com/intel/scikit-learn-intelex/blob/main/scripts/version.py#L35

Reply:
@napetrov @ethanglaser done, please check